Userland/Applications/HexEditor/ValueInspectorModel.h

```diff
@@ -100,6 +103,12 @@ class ValueInspectorModel final : public GUI::Model {
             return "UTF-8";
         case UTF16:
             return "UTF-16";
+        case ASCIIString:
```

Comment:
Hi @konradekk 👋

> not sure if this is really relevant, i guess iʼd go for merging ××× and ×××String values (through case fall-through).

If I understand you correctly, yes I can see how that could work. However, I like the fact that the code explicitly mirrors the view inspector in the UI, like there's a 1-to-1 mapping between each line here and a line in the GUI

> while visiting: maybe we could return “Unknown” for non-matching (i.e. default) case? 🤔

Hmm, the feeling I get from your comment is that `type` (the variable being matched on) is like some dynamic value that we're pattern matching against. Whereas my impression of what's happening here is this is like the definition in code for how the UI looks. Doesn't really seem like we need to even "catch" the default case. Let me know if I've got that wrong

Ah, I do see that there is a `default` option there. And if you're thinking `"Unknown"` would be a better value than `""` there, then... I don't really have an opinion one way or another (haven't thought about where this may even come into play - it's not clear to me we have ever or will really ever hit this).

Reply:
oh! i guess i donʼt see the reason for differentiating `×××` and `×××String` variants then! 😅

definitely out of scope but maybe a yak to shave, even for you, as it seems you still plan to work around that code! 😁

Reply:
Just want to make sure I'm understand you

> oh! i guess i donʼt see the reason for differentiating ××× and ×××String variants then! 😅

So, the xxx and xxxString are two different rows in the value inspector. xxx shows the decoded byte (or word) under cursor, whereas xxxString shows the decoded highlighted area

Do you share that understanding, or are you getting at something else?